sonyapilib/device.py

```diff
@@ -18,6 +18,7 @@
 import wakeonlan
 
 from sonyapilib import ssdp
+from sonyapilib.helper import find_in_xml
```

Comment:
Why not call it `xml` oder `xmlhelper`?

Reply:
if you want to add other helper functions later

Reply:
When adding other helper functions the corresponding file should have a matching name instead of a generic one like `helper` or `util`

Reply:
i thought about putting multiple helper functions in one file. If you want to put your helper functions in multiple files i suggest to put them in a new folder `helper` and then name them for examle `xml.py`

Reply:
Putting multiple unrelated functions into a single file leads always to a file where way too much stuff is put into. Keeping source files small makes it easier to maintain. 
I'd even skip the `helper` folder and just put it on the same level than `device.py` calling it `xml.py`. If at some point a lot of helper file exist a refactoring can be done but I do not think a lot more `helper` files will emerge

Reply:
if it makes you happy ...